luda/editor/client/src/sequenceListView/renderLoadingPage/renderLoadingPage.ts

```diff
@@ -0,0 +1,129 @@
+import {
+  ColorUtil,
+  FontWeight,
+  Language,
+  Rect,
+  Render,
+  Text,
+  TextAlign,
+  TextBaseline,
+  Translate,
+} from "namui";
+import { initTimeline } from "../../timeline/operations/initTimeline";
+import { TimelineSequenceNullableState } from "../../timeline/type";
+import { LoadSequence } from "../operations/loadSequence";
+import { SequenceListViewState } from "../type";
+import { renderCloseButton } from "./renderCloseButton";
+
+export const renderLoadingPage: Render<
+  {
+    sequenceListViewState: SequenceListViewState;
+    timelineState: TimelineSequenceNullableState;
+  },
+  {
+    width: number;
+    height: number;
+  }
+> = ({ sequenceListViewState, timelineState }, props) => {
+  const { loadingSequence } = sequenceListViewState;
+  const loadingSequenceState = loadingSequence?.state;
+
+  if (!loadingSequenceState) {
+    return undefined;
+  }
+
+  const loaded = loadingSequenceState.type === "loaded";
+  const failed = loadingSequenceState.type === "failed";
+
+  if (loaded) {
+    initTimeline(
+      timelineState,
+      loadingSequenceState.tracks,
+      loadingSequence.title,
+    );
+    sequenceListViewState.loadingSequence = undefined;
+  }
```

Comment:
If sequence has been successfully loaded, insert it into the timeline.
And clear loading state